components/mpas-seaice/bld/build-namelist-section

```diff
@@ -28,7 +28,8 @@ add_default($nl, 'config_full_abort_write');
 # Namelist group: decomposition #
 #################################
 
-add_default($nl, 'config_block_decomp_file_prefix', 'val'=>"'${DIN_LOC_ROOT}/ice/mpas-cice/${ICE_GRID}/mpas-cice.graph.info.${date_stamp}.part.'");
+#add_default($nl, 'config_block_decomp_file_prefix', 'val'=>"'${DIN_LOC_ROOT}/ice/mpas-cice/${ICE_GRID}/mpas-cice.graph.info.${date_stamp}.part.'");
+add_default($nl, 'config_block_decomp_file_prefix', 'val'=>"'${DIN_LOC_ROOT}/ice/mpas-cice/${ICE_GRID}/mpas-seaice.graph.v2.${date_stamp}.part.'");
```

Comment:
could you please explain this change? Shouldn't the default be the `graph.info` file since we do not know for sure if the v2 version is available for all meshes?

Reply:
I have reviewed this and will remove it.  I just grepped for graph in the mpas-seaice stuff and made changes in all places that seemed appropriate.  I have now looked at the implementation and agree this should be left unchanged.